### DISCUSSION TITLE
Run Query DSL Without Enabling @Transaction

### DISCUSSION THREAD

**Discussion by sashuprosperoware** [querydsl/querydsl#3019] [Q&A]
I am using querydsl with spring for mysql. The problem is to make any call read/write i need to enable transaction or it wont work. The problem with transaction for reads is it adds to overall time for execution AWS Lambda functions and the functions get teiggered for miions times everyday. Now every millisecond counts. Any help on running at least read calls without transaction is highly appreaciated any idea to change something is also welcomed.

**Comment by jwgmeligmeyling** [comment]
Please see my comments on this thread: https://github.com/querydsl/querydsl/pull/2482#issuecomment-906358161
